externals/tz/CMakeLists.txt

```diff
@@ -1,43 +1,58 @@
 set(TZ_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/tz" CACHE PATH "Time zone source directory")
 set(TZ_DIR "${CMAKE_CURRENT_BINARY_DIR}/tz")
 set(TZ_TMP_SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}/tmpsrc")
-set(TZ_ZONEINFO_DIR "${TZ_DIR}/usr/share/zoneinfo" CACHE PATH "Time zone info data directory")
 set(TZIF_LIST_FILE "${CMAKE_CURRENT_BINARY_DIR}/tzif_list.txt" CACHE PATH "List of zone info files")
+if (TZDB2NX_ZONEINFO_DIR)
+    set(TZ_ZONEINFO_DIR "${TZDB2NX_ZONEINFO_DIR}" CACHE PATH "Time zone info data directory")
+else()
+    set(TZ_ZONEINFO_DIR "${TZ_DIR}/usr/share/zoneinfo" CACHE PATH "Time zone info data directory")
+endif()
 
 find_program(GNU_MAKE make)
-if ("${GNU_MAKE}" STREQUAL "GNU_MAKE-NOTFOUND")
+if (NOT GNU_MAKE)
     message(FATAL_ERROR "GNU make not found")
 endif()
 
+find_program(GIT_PROGRAM git)
+if (NOT GIT_PROGRAM)
+    message(FATAL_ERROR "git program not found")
+endif()
+
 if (NOT EXISTS "${TZ_DIR}" OR NOT EXISTS "${TZIF_LIST_FILE}")
-    # tz's makefile can only build in-tree, so copy the whole source tree to a
-    # separate directory before building.
-    file(COPY ${TZ_SOURCE_DIR}/ DESTINATION ${TZ_TMP_SOURCE_DIR})
-
-    if (APPLE)
-        set(TZ_MAKEFLAGS "LDLIBS=${Intl_LIBRARY}")
-    else()
-        set(TZ_MAKEFLAGS)
-    endif()
+    if (NOT TZDB2NX_ZONEINFO_DIR)
```

Comment:
Maybe it'd be better to use `NOT EXISTS`?

Reply:
No, I specifically want to compare the variable. I'll add a comment there.